docs/source/quickstart.rst

```diff
@@ -0,0 +1,47 @@
+Quickstart
+==========
+
+This guide will show you how to get started using XrViz
+to explore your xarray data.
+
+Installation
+------------
+
+If you are using `Anaconda`_ or Miniconda, install Intake
+with the following commands::
+
+    conda install -c conda-forge xrviz
+    pip install git+https://github.com/intake/xrviz --no-deps
+
+If you are using virtualenv/pip, run the following command::
+
+    pip install git+https://github.com/intake/xrviz
```

Comment:
`pip install xrviz`

You can include the git install, if you like, but call it a development install, and suggest that people install dependencies first.